codeHF/dpl-config_std.json

```diff
@@ -48,8 +48,8 @@
         "d_nSigmaTOFCombined": "5.0"
     },
     "hf-task-d0": {
-        "d_selectionFlagD0": "0",
-        "d_selectionFlagD0bar": "0"
+        "d_selectionFlagD0": "1",
+        "d_selectionFlagD0bar": "1"
```

Comment:
I would suggest to have no selection by default, at least until we have the final set of cuts. It will make bugs easier to spot if we have a stable reference plot throughout the selection development.

Reply:
hi @vkucera. I fixed that in my last PR.